packages/x-codemod/src/v8.0.0/charts/preset-safe/actual.spec.tsx

```diff
@@ -7,9 +7,25 @@ import { ResponsiveChartContainer } from '@mui/x-charts/ResponsiveChartContainer
 import { ChartsOnAxisClickHandler } from '@mui/x-charts/ChartsOnAxisClickHandler';
 import { ChartsXAxis } from '@mui/x-charts/ChartsXAxis';
 import { LegendPosition } from '@mui/x-charts/ChartsLegend';
+import {
+  unstable_useSeries,
+  unstable_usePieSeries,
+  unstable_useLineSeries,
+  unstable_useBarSeries,
+  unstable_useScatterSeries,
+} from '@mui/x-charts/hooks';
+import { unstable_useHeatmapSeries } from '@mui/x-charts-pro/hooks';
 
-// prettier-ignore
-<div>
+function App() {
+  const series = unstable_useSeries();
+  const pieSeries = unstable_usePieSeries();
+  const lineSeries = unstable_useLineSeries();
+  const barSeries = unstable_useBarSeries();
+  const scatterSeries = unstable_useScatterSeries();
+  const heatmapSeries = unstable_useHeatmapSeries();
+
+  // prettier-ignore
+  <div>
```

Comment:
Not sure if we should `return` here to make it look like a normal React component.

Reply:
I guess it doesn't matter, I've never tested if it affects anything though

Reply:
As long as the code got correctly modified, you could also add `<p>I love MUI</p>` at the end the test will be relevant ;)